src/lib/types.ts

```diff
@@ -212,6 +212,21 @@ export interface MakerOracleMessage {
   signature: string;
 }
 
+export interface SoloBridgeTransfer {
+  account: address;
+  perpetual: address;
+  soloAccountNumber: BigNumberable;
+  soloMarketId: BigNumberable;
+  toPerpetual: boolean; // Indicates whether the transfer is from Solo to Perpetual or vice versa.
+  amount: BigNumberable;
+  expiration: BigNumberable;
+  salt: BigNumberable;
```

Comment:
wouldn't expiration and salt not be necessary unless there is a `typedSignature`?
i.e. can't these fields be moved into `SignedSoloBridgeTransfer`?

Reply:
This the same way we did orders, which is there's an `Order` interface and then a `SignedOrder` interface which has a signature.

I tried out adding a `SoloBridgeTransferToBeSigned` interface with `expiration` and `salt`. I think it's a bit too complicated. So instead I made `expiration` and `salt` optional now, let me know what you think